create style

Andimian opened the pull request:
Добавил стили.

Andimian committed:
style

Добавил стили.

Andimian committed:
add img

Andimian committed:
img

прибрался

b-ars07 commented:
1. отступ у последнего элемента - http://prntscr.com/pj7kii
2. от чего отталкиваешься? - http://prntscr.com/pj7ktr используй text-align
3. верхний отступ вылазиет за границу блока - http://prntscr.com/pj7n66
4. в списке, кроме элементов li ничего больше не может быть - http://prntscr.com/pj7noi
5. не по макету - http://prntscr.com/pj7oaq
6. что за разметка - http://prntscr.com/pj7oh1
7. нет контейнера на сайте - http://prntscr.com/pj7pdv
8. что отталкиваешь? - http://prntscr.com/pj7ppw
9. что отталкиваешь? - http://prntscr.com/pj7qcc
10. текст накладывается на картинку и что отталкиваешь? - http://prntscr.com/pj7ql3
11. что отталкиваешь? - http://prntscr.com/pj7r31
12. отталкивать друг друга могут только соседи (про нижний отступ), отступы боковые не нужны. Ну и верхний отступ не понятно для чего - http://prntscr.com/pj7rmv
13. на созвоне говорил про отступы - http://prntscr.com/pj7s5n
14. http://prntscr.com/pj7sd8
15. зачем так много блоков??? - http://prntscr.com/pj7sjx

Andimian committed:
bugs fixed TT week2

margin bugs fixed,
ul bugs fixed
add pading
add conteiner to section
delete not needed blocks

b-ars07 commented:
1. отступа по макету нет - http://prntscr.com/pk89qc
2. не исправлен пункт 7
3. не исправлен пункт 1
4. не исправлен пункт 2
5. не исправлен пункт 3

Не вижу исправлений на хостинге

Andimian committed:
update

update style and run
git push origin week2:gh-pages

Andimian committed:
Merge branch 'master' into week2

b-ars07 commented:
1. говорил про отступы на созвоне. Отталкиваем мы нижний от верхнего, через марджин - http://prntscr.com/pknnn3
2. http://prntscr.com/pknphx
3. у элементов одного списка должна быть одна разметка. блок див - лишний - http://prntscr.com/pknpz1

Andimian committed:
upgrade

Удалил див. который я использовал как фон у активной страницы пагинации.
Убрал марджин топ у списка и марджин ботом у пагинации, добавил марджин ботом всему блоку ласт ньюс.

Andimian committed:
Merge branch 'week2' of https://github.com/Sokolov45/loftshool into week2

Andimian committed:
update

поправил название классов
